index.html

```diff
@@ -18,9 +18,9 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <div class="centering-outer-container">
-    <div class="centering-inner-container">
-      <div id="content" class="centering-content">
+  <div class="page-container">
+    <div class="content-box">
+      <div id="content">
```

Comment:
I kept the `div id="content"` because I'm going to use it in my next PR to enable you to easily change its displayed content.